develop/smart-contracts/libraries/index.md

```diff
@@ -1,9 +1,25 @@
 ---
 title: Libraries
-description: TODO
-hide: 
-    - feedback
+description: Discover libraries for interacting with smart contracts on Polkadot, including Ethers.js, Web3.js, viem, and other essential tools.
 template: index-page.html
 ---
 
-# Libraries
\ No newline at end of file
+# Libraries
+
+Leverage popular libraries to interact with smart contracts on Polkadot-based networks. These libraries simplify contract calls, event listening, and transaction handling.
+
+## Selection Guide
+
+When choosing a library for your project, consider:
+
+- **Development experience** – some libraries offer more intuitive APIs for specific programming backgrounds
+- **Project requirements** – evaluate performance needs, bundle size, and specific feature support
+- **Community support** – larger communities often mean better documentation and more examples
```

Comment:
Maybe we do a table again here 🤷‍♀️

Reply:
I like the info we're providing users like we're telling them the things they should consider when picking a library, but we're not providing the actual details and that's where maybe it's not our place, but maybe we should provide very objective concrete things to compare, like viem is more modular and typescript friendly than ethers object oriented approach 🤷‍♀️ 🤷‍♀️